tests/NTest_adc_env.lua

```diff
@@ -4,21 +4,23 @@
 local N = ...
 N = (N or require "NTest")("adc-env")
 
--- TODO: Preflight test that we are in the correct environment with an I2C
--- expander in the right place with the right connections.
+local NTE = require "NTestEnv"
 
 -- TODO: Use the mcp23017 module in the main tree rather than hand-coding
 -- the commands
 
 N.test('setup', function()
-  -- Configure the ADC
+  -- Ensure that we're on DUT 0
+  assert(NTE.getFeat('DUT') == 0, "Not on DUT 0")
+
+  -- Configure the ADC (this implicitly checks for having the ADC module)
```

Comment:
Early checks with distinct error message are a nice thing to have. So since we now have respective methods we should use them. Maybe even have such checks for basic modules as sjson before using them in `NTestEnv`